test/odbc/psqlodbc/test_pg_datatype_bigint.cpp

```diff
@@ -0,0 +1,1016 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "../src/drivers.h"
+#include <cmath>
+#include <iostream>
+#include <time.h>
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.bigint_table_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.bigint";
+const string VIEW_NAME = "master_dbo.bigint_view_odbc_test";
+vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "INT PRIMARY KEY"},
```

Comment:
Why was this changed to "INT"? The original tests used Big int datatype for primary key?

Reply:
Consistency with other tests.
The Primary key in most tests are inserted using a for loop's index and with some tests, using the test type wouldn't work eg DateTime

Reply:
For some of those types yes, but would be nice to test the primary key constraint of some of these types as well, but not a hard requirement.